Automattic/jetpack
# Anchor: Update post-publish prompt text.

This PR updates the post-publish "Convert to audio" text to reference Anchor.


| Before | After |
| --- | --- |
<img width="337" alt="Screen Shot 2021-02-05 at 10 58 50 AM" src="https://user-images.githubusercontent.com/349751/107077095-3adeaf00-67a1-11eb-9f71-b74add5b05c6.png"> | <img width="342" alt="Screen Shot 2021-02-05 at 10 46 15 AM" src="https://user-images.githubusercontent.com/349751/107076328-23eb8d00-67a0-11eb-9af5-fe50bfc22ec7.png">

Fixes #461-gh-Automattic/dotcom-manage

#### Changes proposed in this Pull Request:

* text only

#### Jetpack product discussion
* pbAPfg-PF-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Enable Experimental extensions.
* Publish a post.
* Verify the post-publish prompt matches the above screenshot.

#### Proposed changelog entry for your changes:
* None needed.
